default/python/AI/ShipDesignAI.py

```diff
@@ -1018,7 +1032,7 @@ def _class_specific_filter(self, partname_dict):
         pass
 
     def optimize_design(self, additional_parts=(), additional_hulls=(),
-                        loc=None, verbose=False, consider_fleet_count=True):
+                        loc=None, verbose=False, consider_fleet_count=False):
```

Comment:
You change default value, but did not change usages. Looks like improvement for me.